src/main/java/com/terraformersmc/campanion/recipe/TarpRecipe.java

```diff
@@ -26,9 +27,9 @@ public boolean matches(CraftingInventory inv, World world) {
 		int shearAmount = 0;
 		for (int i = 0; i < inv.size(); i++) {
 			ItemStack stack = inv.getStack(i);
-			if(FabricToolTags.SHEARS.contains(stack.getItem())) {
+			if(stack.isIn(CampanionItemTags.SHEARS)) {
```

Comment:
We don't need our own shears tag. Please use `net.fabricmc.fabric.api.tag.convention.v1.SHEARS`